modules/webui.nix

```diff
@@ -4,123 +4,11 @@ with lib;
 
 let
   cfg = config.contrail.webui;
+  confFile = import (./configuration + "/R${contrailPkgs.contrailVersion}/webui.nix") { inherit pkgs cfg contrailPkgs; };
+
   web-server = pkgs.writeTextFile {
```

Comment:
to remove

Reply:
Really? Shouldn't there be some default configuration?

Reply:
Ah sorry, you are probably referring to the empty line hehe